src/content/docs/discovery/index.mdx

```diff
@@ -0,0 +1,98 @@
+
+## Identify extensions
+
+Before starting the project, use the following list to create an inventory of the Adobe Commerce extensions that are actively being used. This will help you understand which extensions can be replaced by out-of-the-box Adobe Commerce functionality.
+
+* What extensions are currently in use?
+
+* What type of data do the extensions provide (for example, reviews)?
+
+* Is the data required on the frontend?
+    
+* How do the extensions expose the data (for example, GraphQL, REST API)?
+
+* Do the extensions expose an API to access data (for example, [Amasty Labels](https://amasty.com/docs/doku.php?id=magento_2:product_labels))? If not, create an action item to expose the required data through an API. Options include:
+
+  * Customize the Adobe Commerce Catalog Service exporter to export additional custom data to Catalog Service
+  * Create a custom Adobe Commerce GraphQL query
+  * Use [API Mesh for Adobe Developer App Builder](https://developer.adobe.com/graphql-mesh-gateway/)
+
+* Are any of the extensions for delivery options (for example, shipping/BOPIS), payments, or tax providers? If you use third-party solutions, clarify if they expose APIs on the frontend and if they provide their own set of dropin components for the frontend integration.
+
+## Validate option for dropins/widgets
+
+Dropins are re-useable components that define the storefront shopping experience. They are framework agnostic and can be used in any context (Edge Delivery Services, AEM, Luma). However, this documentation focuses on the use of dropins in Edge Delivery Services projects using the [boilerplate template](https://github.com/hlxsites/aem-boilerplate-commerce). The dropin development roadmap is synchronised with Adobe Commerce APIs, so new API features are automatically available in dropins and widgets.
+
+Storefront widgets in Adobe Commerce Live Search are tools that enhance the search functionality on your site. Widgets are designed to improve the user experience by making search results more accessible and relevant. There are two main types of widgets:
+
+* **[Live Search Popover](https://experienceleague.adobe.com/en/docs/commerce-merchant-services/live-search/live-search-storefront/storefront-popover)**: This widget appears as a box under the search field, displaying search results dynamically as users type their queries.
+* **[Product Listing Page (PLP)](https://experienceleague.adobe.com/en/docs/commerce-merchant-services/live-search/live-search-storefront/plp-styling)**: This widget provides a searchable product listing page with facets and synonym support, making it easier for users to find products.
+
+:::note
+See the [dropins overview](/dropins/all/introduction/) for a list of all available dropins.
+:::
+
+### Use cases and requirements
+
+Document your use cases and requirements. If there are gaps, the Adobe team can help you understand how to fill them. For example:
+
```

Comment:
Replace 81 with

If you see gaps in what these components support, Adobe team can help with the plan to achieve your scenarios. for example: